creator/.flutter-plugins

```diff
@@ -1,11 +1,25 @@
 # This is a generated file; do not edit or check into version control.
```

Comment:
As it says here, maybe it is better to add this file to gitignore (remove it from version control). This is the recommended gitignore configuration for flutter: https://github.com/flutter/flutter/blob/master/.gitignore

Reply:
OK I'll make a new  issue